doc/source/conf.py

```diff
@@ -294,6 +296,17 @@
 autodoc_typehints = "signature"
 
 
+def filter_out_undoc_class_members(member_name, class_name, module_name):
+    module = import_module(module_name)
+    cls = getattr(module, class_name)
+    if getattr(cls, member_name).__doc__:
+        return f"~{class_name}.{member_name}"
```

Comment:
One addition we can make:

Check if it's a classmethod/staticmethod and remove the `~` in that case.

Reply:
Decision: We will do this in a future PR.